.github/ISSUE_TEMPLATE/UPDATE_ICON.yml

```diff
@@ -1,7 +1,7 @@
 name: Propose updates to an existing icon
 description: For modifications to an existing icon in polaris-icons
 title: '[Icon]: Update icon <icon-name>'
-labels: ['Icon', 'untriaged', 'Fast follow']
+labels: ['Icon', 'untriaged', 'Uplift polish']
```

Comment:
I don't know if that should be an automatic added label I would ask @lgriffee. Not blocking but might be worth checking.

Reply:
```suggestion
labels: ['Icon', 'untriaged', '#gsd:38846']
```

I haven't been tracking the 'Uplift polish' labels anymore and have shifted to using the one above. We just need to make sure this gets updated as the project labels change in the future. Also is there a way to add these issues to the Backlog automatically? That would also be helpful!

Reply:
I guess I am unsure if all new icon requests should go to this project. That doesn't feel right to me.

cc @travis-eh

Reply:
Whoops I didn't refresh this page before merging so I didn't see this comment before merging.
I updated it so that templates to add or update an icon only have the `Icon` and `untriaged` labels.
I can update this in a follow up PR if there's another label we'd like to include 👍